### PR TITLE
Cas fixes javadoc issues

### PR DESCRIPTION
**Pull request by Cas-B**
Perform the Code Review as mentioned in the document: Code-Review-How-To.odt

### CODE REVIEW

File: src/main/java/org/group17/Game/Game.java [hunk 1]
```diff
@@ -166,21 +166,21 @@ public GameMap getMap() {
...
-	
+
 	/**
 	 * Method to win the game.
 	 */
 	public void gameWon() {
 		GUI.getInstance().gameHasBeenWon();
 	}
-	
+
```
Comment (kusterlu) [inline src/main/java/org/group17/Game/Game.java]: changes to Game reviewed. naming cosistency, <layout,

File: src/main/java/org/group17/Game/Logic/EndingHook.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
 public class EndingHook implements GameHook {
 
 	/**
-	 * This method runs ending hook every iteration.
+	 * {@inheritDoc}
```
Comment (kusterlu) [inline src/main/java/org/group17/Game/Logic/EndingHook.java]: good to know @inheritDoc is possible! <layout,

File: src/main/java/org/group17/Gui/EndPanel.java [hunk 3]
```diff
@@ -19,13 +18,14 @@
 public class EndPanel extends JPanel {
 
 	private Constants cConstants = new Constants();
+	
 	/**
-	 * help window sketched.
-	 * @param i background whether won or lost.
+	 * Creates the EndScreen.
```
Comment (kusterlu) [inline src/main/java/org/group17/Gui/EndPanel.java]: formal rephrase acknowledged. <layout,

File: src/main/java/org/group17/Gui/EndPanel.java [hunk 4]
```diff
@@ -47,7 +47,7 @@ public void initExitButton() {
 		jbtBack.setContentAreaFilled(false);
 		jbtBack.addActionListener(new ActionListener() {
 			public void actionPerformed(ActionEvent e) {
-				Main.getGUI().getFrame().dispose();
+				GUI.getInstance().getFrame().dispose();
```
Comment (kusterlu) [inline src/main/java/org/group17/Gui/EndPanel.java]: GUI is the shortest access point indeed. Had the same search today. <layout,

File: src/main/java/org/group17/Gui/EndPanel.java [hunk 5]
```diff
@@ -55,22 +55,22 @@ public void actionPerformed(ActionEvent e) {
...
 	/**
-	 * Method for setting backround on lost.
+	 * Method for setting background when the game has been lost.
 	 */
 	public void setLoosingBackground() {
-		ImageIcon endIcon = new ImageIcon(cConstants.getImageOfEndLoosing());
+		ImageIcon endIcon = new ImageIcon(this.cConstants.getImageOfEndLoosing());
 		JLabel endPic = new JLabel(endIcon);
-		endPic.setBounds(0, 0, cConstants.getRes()[0], cConstants.getRes()[1]);
+		endPic.setBounds(0, 0, this.cConstants.getRes()[0], this.cConstants.getRes()[1]);
```
Comment (kusterlu) [inline src/main/java/org/group17/Gui/EndPanel.java]: patient work with all the this <layout,

File: src/main/java/org/group17/Gui/GamePanel.java [hunk 6]
```diff
@@ -41,50 +37,27 @@ public void setUpdateThread(Thread tUpdateThread) {
 	}
 
 	/**
-	 * returns the associated Game.
```
Comment (kusterlu) [inline src/main/java/org/group17/Gui/GamePanel.java]: was this truely redundant? <layout,

File: src/main/java/org/group17/Gui/WelcomePanel.java [hunk 7]
```diff
@@ -49,8 +48,8 @@ public void initStartButton() throws Exception {
 		startButton.setContentAreaFilled(false);
 		startButton.addActionListener(new ActionListener() {
 			public void actionPerformed(ActionEvent e) {
-				Main.getGUI().removeWelcomePanel();
-				Main.getGUI().setGameWindow();
+				GUI.getInstance().removeWelcomePanel();
```
Comment (kusterlu) [inline src/main/java/org/group17/Gui/WelcomePanel.java]: It's great you removed all the getGUI dependencies @Cas-B , so we can delete the method from the Main, as told so. <layout,

File: src/main/java/org/group17/Gui/KeyListen.java [hunk 8]
```diff
@@ -48,24 +48,24 @@ public void run() {
...
+				this.timer.purge();
 			}
 			break;
 		case KeyEvent.VK_SPACE:
-			controlHook.setBubble(true);
+			this.controlHook.setBubble(true);
 			break;
 		case KeyEvent.VK_UP:
-			controlHook.setJump(true);
+			this.controlHook.setJump(true);
```
Comment (kusterlu) [inline src/main/java/org/group17/Gui/KeyListen.java]: My god, SO many this statements ;) <layout,

File: src/main/java/org/group17/Map/GameMap.java [hunk 9]
```diff
@@ -134,7 +134,7 @@ public void drawScore(Graphics gGraphics) throws Exception {
 	}
 	
 	/**
-	 * 
+	 * Draws the amount of lives in the screen.
```
Comment (kusterlu) [inline src/main/java/org/group17/Map/GameMap.java]: reviewed GameMap : this statements and Javadoc. <layout,

File: src/main/java/org/group17/Units/Enemy.java [hunk 10]
```diff
@@ -54,7 +54,7 @@ public void onCollisionWith(GameObject goOther) {
 			this.bInBubble = true;
 		} else {
 			this.getSpeed().setX(-this.getSpeed().getX());
-			if (goOther instanceof Player && bInBubble) {
+			if (goOther instanceof Player && this.bInBubble) {
```
Comment (kusterlu) [inline src/main/java/org/group17/Units/Enemy.java]: this method is rephrased meanwhile and must be fixed manualy. <layout,

**Comment by kusterlu** [comment]
I've run mvn site and nothing complains.

**Comment by kusterlu** [comment]
This can be merged, I made an extremely thorough review. 
Please resolve merge-conflics.

File: src/main/java/org/group17/Game/Logic/EndingHook.java (hunk 2 above)
Comment (inheritdoc) [inline src/main/java/org/group17/Game/Logic/EndingHook.java]: With @inheritdoc, anything is possible

**Comment by kusterlu** [comment]
haha. it give me 25 build errors. what the hell! <bug,

**Pull request merged**